src/pages/ChangePassword.tsx

```diff
@@ -0,0 +1,194 @@
+import { useState } from 'react';
+import { useForm, SubmitHandler } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { ChangePasswordSchema } from '../validations/ConfirmPasswordValidation';
+import {
+  Container,
+  Form,
+  StyledIconOpenEyes,
+  StyledIconClosedEyes,
+  Button,
+  Checklist,
+  List,
+  ErrorMessages,
+  MessageChecklist,
+  ChangePasswordErrorMessage,
+  Border,
+} from './styles/ChangePasswordStyles';
+import { useApi } from '../hooks/useApi';
+import Header from '../components/Portal/Header';
+import Footer from '../components/Portal/Footer';
+import ChangePasswordModal from '../components/Portal/ProfileModal/ChangePasswordModal'; 
+import CancelModal from '../components/Portal/ProfileModal/CancelModal';
+
+
+interface ChangePasswordForm {
+  oldPassword: string;
+  password: string;
+  confirmNewPassword: string;
+}
+
+const ChangePassword = () => {
+  const [oldVisible, setOldVisible] = useState(false);
+  const [visible, setVisible] = useState(false);
+  const [confirmVisible, setConfirmVisible] = useState(false);
+  const [errorMessage, setErrorMessage] = useState<string>();
```

Comment:
Aqui ela está tipada como string, não deveria começar com uma string vazia?
```suggestion
  const [errorMessage, setErrorMessage] = useState<string>('');
```